app/views/school/internet/mobile/extra_data_requests/guidance.html.erb

```diff
@@ -0,0 +1,55 @@
+<%- content_for :before_content do %>
+  <%- school_breadcrumbs(items: [
+      { t('page_titles.school.internet.home.show.title') => internet_school_path(@school) },
+      t('page_titles.request_extra_mobile_data')
+    ],
+    user: @user,
+    school: @school) %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">
+      <%= t('page_titles.request_extra_mobile_data') %>
+    </h1>
+
+    <h2 class="govuk-heading-m">1. Check who is eligible for extra mobile data</h2>
+
+    <p class="govuk-body">You can request extra mobile data for children who:</p>
+
+    <ul class="govuk-list govuk-list--bullet">
+      <li>have no broadband connection at home</li>
+      <li>cannot afford additional data</li>
+    </ul>
+
+    <p class="govuk-body">You can make up to 10 requests for <%= govuk_link_to 'specific circumstances', specific_circumstances_school_path %>.</p>
```

Comment:
Is it worth extracting the 10 into a constant, maybe on the `ExtraMobileDataRequest` model?